src/LibLog/LibLog.cs

```diff
@@ -568,8 +625,10 @@ public bool Log(LogLevel logLevel, Func<string> messageFunc, Exception exception
                 return false;
             }
         }
+#endif
     }
 
+#if !LIBLOG_PROVIDERS_ONLY
```

Comment:
The final API has no `ILog`. I think providing one, and this, would just be noise. Ultimately the purpose of LibLog in the providers only mode is purely to allow an existing logging infra to use LibLog clever support for logging frameworks. This should all be taken care of at a higher level in the that logging infra.